src/test/resources/db/pre-populate-inn-reach-location-code.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO inn_reach_location (id, code, description, created_by, created_date, last_modified_by, last_modified_date)
```

Comment:
Which test is this file used in?

Reply:
I used it just for the local test, but it will be used in REST API integration tests